src/test/java/com/wikia/webdriver/pageobjectsfactory/componentobject/editprofile/AvatarComponentObject.java

```diff
@@ -7,13 +7,17 @@
 import org.openqa.selenium.support.FindBy;
 
 import java.io.File;
+import java.util.List;
+import java.util.Random;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'java.util.Random'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AUselessImportCheck)